consensus/clique/clique.go

```diff
@@ -347,6 +357,15 @@ func (c *Clique) verifyCascadingFields(chain consensus.ChainReader, header *type
 	if parent.Time.Uint64()+c.config.Period > header.Time.Uint64() {
 		return ErrInvalidTimestamp
 	}
+	// Verify that the gas limit remains within allowed bounds
+	diff := int64(parent.GasLimit) - int64(header.GasLimit)
+	if diff < 0 {
+		diff *= -1
```

Comment:
`diff = -diff` maybe quicker.

Reply:
Maybe you're right, but I think it's very small optimization. I don't want to modify the original author's code. In the future people may be confused why they are different.